data/policies/extend.cas

```diff
@@ -0,0 +1,15 @@
+resource foo {
+	allow(bar, foo, file, getattr);
+}
+
+extend foo {
+	fn my_func(domain source) {
```

Comment:
I'd like to see a (failed) test with the same function being defined in the original resource and the extended one.

Reply:
It turns out this is broken regardless of being in an "extend" block or not.  (There's a FIXME note in compile.rs about it).

I'm working on an update to handle it pretty generically, at AliasMap insertion time, but there are quite a few different cases, notably if one of the duplicated names was synthetically created rather than being explicitly declared, we need to take some care in making sure that we write a clear error message to explain to the user why there is a conflict.

So I'm beginning to wonder if it would make sense to handle that bug as a separate task since it could probably use some more focused attention in a PR to make sure all the cases are handled appropriately.

Reply:
Actually, still not quite working.  I'll aim to push a second commit tomorrow AM.

Reply:
Pushed